module/surveys/helper.js

```diff
@@ -1570,9 +1570,9 @@ module.exports = class SurveysHelper {
             }
             let currentDate =new Date()
             if (
-              solutionData.data.hasOwnProperty("endDate") &&
-              new Date(solutionData.data.endDate) <  currentDate
-
+              solutionData.data.hasOwnProperty('endDate') &&
+              new Date(solutionData.data.endDate) < currentDate &&
+              solutionData.data.referenceFrom !== messageConstants.common.PROJECT
```

Comment:
@borkarsaish65 avoiding expiry check if survey is reference from project. This change suffice for all requirements of stage 2? no change required for observations? what about changes in project service. please reply here